### DISCUSSION TITLE
discord hacked

### DISCUSSION THREAD

**Discussion by Gammagamesprods** [TechItAllServices/TechItAllServices-Main#2] [General]
hey my discord is hacked but i might recover it, @TechItAll can you monitor it until i do? more info in the ptd

**Comment by TechItAll** [comment]
Ok...? I will look over it. I will also isolate your discord in my server.

**Reply by Gammagamesprods** [reply]
ok thx

**Reply by TechItAll** [reply]
I'm super confused tho.

**Reply by Gammagamesprods** [reply]
also why the hell am i in the ptd il even show u the project me and parker are working on

**Reply by Gammagamesprods** [reply]
my password and email wont work in sign in

**Reply by TechItAll** [reply]
Oh, that was before I knew who you were. Parkers nit the smartest and made it sound very strange. I forgot you were even in there. Sry.

**Reply by TechItAll** [reply]
Can you make a new account?

**Reply by Gammagamesprods** [reply]
oh its ok and he thinks my dads a teacher but hes actually a professor at ub, not a teacher here

**Reply by Gammagamesprods** [reply]
im trying to recover this one first but yeah i did but now its saying i need a phone number but i dont have one

**Reply by TechItAll** [reply]
Ok. So do you plan on recovering your discord or making a new one?

**Reply by TechItAll** [reply]
Oh...that's a issue.

**Reply by Gammagamesprods** [reply]
both, i already made a new one but now i can use it because it says i need a phone verification but i dont have a phone

**Reply by Gammagamesprods** [reply]
yeah, it is

**Reply by TechItAll** [reply]
Idk what you can do.

**Reply by Gammagamesprods** [reply]
do u know about a virtual phone number i can use in my browser or somthing?

**Reply by TechItAll** [reply]
Yes but also no. You can use Google voice but....... you need to set that up with a phone number.

**Reply by Gammagamesprods** [reply]
oh i tried that and its blocked, AAAAAAUUUUUUUUGGGGGGGHHHHH

**Reply by Gammagamesprods** [reply]
and everything else costs monee and i dont have any

**Reply by Gammagamesprods** [reply]
also i saw somthings about a terminal but i cant find it can u show me where it is???

**Reply by Gammagamesprods** [reply]
or can i even access it? is it another org or somthin?

**Reply by TechItAll** [reply]
org?

**Reply by Gammagamesprods** [reply]
organization

**Reply by TechItAllSA** [reply]
I don't know. I would just try to make a new account at this point.

**Reply by Gammagamesprods** [reply]
yeah im gonna do it now my freind is bringing his phone to class so il do it now

**Reply by Gammagamesprods** [reply]
YESSSSSSSS I GOT IT TO WORK
the username is the_real_gamma_games

**Reply by TechItAll** [reply]
Ok

**Comment by TechItAll** [comment]
Issue Has Been Solved!
